server/src/main/java/org/elasticsearch/action/bulk/TransportBulkAction.java

```diff
@@ -394,37 +406,46 @@ private long buildTookInMillis(long startTimeNanos) {
      * retries on retryable cluster blocks, resolves item requests,
      * constructs shard bulk requests and delegates execution to shard bulk action
      * */
-    private final class BulkOperation extends ActionRunnable<BulkResponse> {
```

Comment:
Removing `ActionRunnable` here felt more important when we optionally forked. Now that we've mostly removed that I don't think I *need* to remove this. But I think it is more clear without it.